### PR TITLE
Update error status from 500 to 403

### PR DESCRIPTION
**Pull request by obutterbach**

### CODE REVIEW

**Comment by obutterbach** [comment]
I ran `npm test` on this change, just to make sure tests are not broken

File: app/proxy.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const tunnel = httpProxy.createProxyServer(config.tunnel);
 function onError(error, req, res) {
   log.error(error);
 
-  res.status(error.status || 500);
+  res.status(error.status || 403);
```
Comment (r4vi) [inline app/proxy.js]: this is changing the default error code from 500 to 403. Have a look at https://github.com/ministryofjustice/analytics-platform-auth-proxy/blob/cdb7fd363f904785e3ef8373c60565eee12fe568/app/base/handlers.js#L86

That's the function that creates the Error.

You can scroll down to `portAllowed` in the same file and see how that sets the `error.status`

**Comment by r4vi** [comment]
there are no tests for this right now so nothing will break... would be good if you can add them.  Have a look at https://github.com/ministryofjustice/analytics-platform-auth-proxy/blob/ba03977e7958f49ec84860640fd7063fa61a1791/app/__tests__/middleware/rstudio.js to see how another middleware is tested

File: app/proxy.js (hunk 1 above)
Comment (obutterbach) [reply app/proxy.js]: Oh I see! thanks a lot :)

**Comment by obutterbach** [comment]
Ready for a new review

File: app/__tests__/middleware/proxy.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+});
+
+describe("Test proxy response", () => {
+  test("It should GET 403", async () => {
+    const app = create_app();
+    const response = await request(app).get("/");
+    expect(response.statusCode).toBe(403);
+    expect(response.headers.location).toBe(undefined);
+  });
+  test("It should a 403 on login", async () => {
```
Comment (r4vi) [inline app/__tests__/middleware/proxy.js]: I think you can remove this one? as it doesn't 403 on login

Comment (obutterbach) [reply app/__tests__/middleware/proxy.js]: Ah yeah, you're right 
Let me correct this

**Pull request merged**